### ISSUE TITLE
Chinese input problem in the ipad

### ISSUE THREAD

**Issue by plusjhjhjh** [remnoteio/remnote-issues#257]
**Describe the bug**
When i try to type something in the ipad's safari,after i tap the enter the character i have input,the cursor will unexpectly return to the first location that makes me can't type fluently

**Comment by mfranzs** [comment]
Thanks for reaching out. Are you in desktop mode here? Is there a way for you to switch into mobile mode?

**Comment by plusjhjhjh** [comment]
> Thanks for reaching out. Are you in desktop mode here? Is there a way for you to switch into mobile mode?

Yes,it happened in the desktop mode.Because of it ,i can only use the mobile mode to type Chinese
